app/logic/events.py

```diff
@@ -507,6 +508,79 @@ def getEventRsvpCount(eventId):
     """
     return len(EventRsvp.select().where(EventRsvp.event_id == eventId))
 
+def getCountdownToEvent(event, *, currentDatetime=None):
+    """
+    Given an event, this function returns a string that conveys the amount of time left
+    until the start of the event.
+
+    Note about dates:
+    The way dates are communicated is unintuitive. There are two major rules that govern how we discuss dates.
```

Comment:
I think we could remove this line from the docstring. I think it can be a little confusing what you are talking about when you say "the way dates are communicated is unintuitive", plus the docstring flows just fine without it.

Reply:
Fixed!